docs/json_rpc_api.md

```diff
@@ -1234,10 +1237,10 @@ func main() {
 Example output:
 
 ```
-2014/05/12 20:33:17 Client shutdown in 10 seconds...
-2014/05/12 20:33:19 Block connected: 000000000000000007dff1f95f7b3f5eac2892a4123069517caf34e2c417650d (300461)
-2014/05/12 20:33:27 Client shutting down...
-2014/05/12 20:31:27 Client shutdown complete.
+2018/01/13 16:14:30 Client shutdown in 10 seconds...
```

Comment:
For consistency, if you decide to remove the timestamps from the others, please remove them from these as well.

Reply:
OK